pulsar/consumer_partition.go

```diff
@@ -1265,6 +1265,10 @@ func (pc *partitionConsumer) _getConn() internal.Connection {
 	return pc.conn.Load().(internal.Connection)
 }
 
+func (pc *partitionConsumer) messagesInQueue() int {
+	return len(pc.queueCh)
```

Comment:
If there are multi go routines consuming and producing to this channel this value might not be useful after being read.